lua/rustaceanvim/dap.lua

```diff
@@ -170,6 +230,9 @@ function M.start(args)
       -- create debug configuration
       local dap_config = {
         name = 'Rust tools debug',
+
+        -- TODO: arguably, this type should just be 'lldb`. Is there an important reason to
+        -- use a custom type for the plugin?
         type = 'rt_lldb',
```

Comment:
I'm not entirely sure (this was taken over from rust-tools.nvim).
I believe it may be to avoid overriding existing dap `rust` or `lldb` configurations.

Perhaps a better approach to this would be to load `dap.configurations.rust` and to extend it with the discovered configurations, and then to query nvim-dap for configurations to run.
That is [how I do it in haskell-tools.nvim](https://github.com/mrcjkb/haskell-tools.nvim/blob/dc778953e98bf2667424d93dad82eefe319e0f7d/lua/haskell-tools/dap.lua#L144C54-L150C8).

But maybe this is for another PR. I'll create an issue for it :smile:

Reply:
Created https://github.com/mrcjkb/rustaceanvim/issues/66